extension-manifest-v3/src/utils/offscreen.js

```diff
@@ -9,13 +9,14 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0
  */
 
-import { define } from 'hybrids';
+const permissionRequest = {
+  permissions: ['offscreen'],
+};
 
-// Globals
-import '../panel/index.js';
-
-// Components
-define.from(import.meta.glob('./**/*.js', { eager: true, import: 'default' }), {
-  prefix: 'ui-settings',
-  root: 'components',
-});
+export async function requestPermission() {
+  if (!(await chrome.permissions.contains(permissionRequest))) {
+    if (!(await chrome.permissions.request(permissionRequest))) {
+      throw new Error('Ghostery requires "offscreen" permission');
```

Comment:
Should this throw if both are false (like it is now) or if one of condition is not met?